app/views/layouts/volunteer_notification.html.haml

```diff
@@ -0,0 +1,41 @@
+!!!
+%html{:xmlns => "http://www.w3.org/1999/xhtml"}
+  %head
+    / If you delete this tag, the sky will fall on your head
```

Comment:
Sometimes, some mail browsers do things we don't expect. Suggest removing any/all comments.